vm/src/obj/objint.rs

```diff
@@ -192,13 +192,27 @@ fn inner_divmod(int1: &BigInt, int2: &BigInt, vm: &VirtualMachine) -> PyResult {
 }
 
 fn inner_lshift(int1: &BigInt, int2: &BigInt, vm: &VirtualMachine) -> PyResult {
-    let n_bits = get_shift_amount(int2, vm)?;
-    Ok(vm.ctx.new_int(int1 << n_bits))
+    inner_shift(int1, int2, |a, b| a << b, vm)
 }
 
 fn inner_rshift(int1: &BigInt, int2: &BigInt, vm: &VirtualMachine) -> PyResult {
-    let n_bits = get_shift_amount(int2, vm)?;
-    Ok(vm.ctx.new_int(int1 >> n_bits))
+    inner_shift(int1, int2, |a, b| a >> b, vm)
+}
```

Comment:
we don't need one line `inner_lshift` and `inner_rshift` anymore. caling `inner_shift` in each shift funciton will be enough.